src/core/Window.cpp

```diff
@@ -323,15 +323,14 @@ void Window::InitializeWindowManager(std::string_view gfxBackend, std::string_vi
 #endif
 #if defined(ENABLE_OPENGL) || defined(__APPLE__)
     if (gfxBackend == "sdl") {
-        if (gfxApi == "OpenGL") {
```

Comment:
I removed it because it breaks old json compatibility for people that dont have `GfxAPI` stored in their json file and causes a crash. Basically if you are set to SDL it should always default to use opengl. Only if you are apple and have it set to metal does it change to metal instead.

There are only two sdl apis, OpenGL and now Metal. There needs to be a default, why wouldn't it be OpenGL?